fix(macos): Fix incorrect Redo and CloseWindow accelerators

probablykasper opened the pull request:
Did not understand if I need to make a change file or not

<!--
Please make sure to read the Pull Request Guidelines:
https://github.com/tauri-apps/tauri/blob/dev/.github/CONTRIBUTING.md#pull-request-guidelines
-->

<!-- PULL REQUEST TEMPLATE -->
<!-- (Update "[ ]" to "[x]" to check a box) -->

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)
<!--
If yes, please describe the impact and migration path for existing applications in an attached issue. Filing a PR with breaking changes that has not been discussed and approved by the maintainers in an issue will be immediately closed.
-->

- [ ] Yes. Issue #___
- [x] No


**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

probablykasper committed:
fix(macos): Fix incorrect Redo accelerator

lemarier reviewed (changes requested):
Can you add a changefile please

also would be great as you are there if you can check https://github.com/tauri-apps/tao/issues/159

thanks

probablykasper committed:
fix(macos): Fix incorrect CloseWindow accelerator (#159)

probablykasper commented:
Changed `CloseWindow` to `CMD+W` and added the changefile (Did not understand when change files are needed, maybe the wording of that should be changed?)
